java-cfenv/src/main/java/io/pivotal/cfenv/core/CfCredentials.java

```diff
@@ -15,11 +15,7 @@
  */
 package io.pivotal.cfenv.core;
 
-import java.util.ArrayList;
-import java.util.Arrays;
-import java.util.HashMap;
-import java.util.List;
-import java.util.Map;
+import java.util.*;
```

Comment:
```suggestion
import java.util.ArrayList;
import java.util.Arrays;
import java.util.HashMap;
import java.util.List;
import java.util.Map;
import java.util.Objects;
```
The preference would be for individual imports rather than the whole package - is there a reason you chose to make this change?